en/docs/reference/product-apis/service-catalog-apis/service-catalog-v1/service-catalog-v1.yaml

```
@@ -0,0 +1,1089 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInfoList'
+          headers:
+            ETag:
+              $ref: '#/components/headers/ETag'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+              -F "file=@services.zip" "https://127.0.0.1:9443/api/am/service-catalog/v0/services/import?overwrite=true"'
+                
+  #----------------------------------------------------------
+  # Exporting a service as a zipped archive
+  #----------------------------------------------------------
+  /services/export:
+
+    #-----------------------------------------
+    # Retrieve a service as a zipped archive
+    #-----------------------------------------
+    get:
+      summary: Export a service
+      description: |
+        Export a service as an archived zip file.
+      security:
+        - OAuth2Security:
+            - service_catalog:service_view
+      operationId: exportService
+      tags:
+        - Services
+      parameters:
+        - name: name
+          in: query
+          description: |
+            Name of the service to export
+          required: true
+          schema:
+            type: string
+        - name: version
+          in: query
+          description: |
+            Version of the service to export
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: |
+            Successful response as the exported service as a zipped archive.
+          content:
+            application/zip:
+              schema:
+                type: string
+                format: binary
+          headers:
+            ETag:
+              $ref: '#/components/headers/ETag'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/service-catalog/v0/services/export?name=Single1Curl&version=1.0.0" > services.zip'
+
+  ######################################################
+  # The Service Catalog API settings
+  ######################################################
+  /settings:
+    #-----------------------------------------------------
+    # Retrieve Service Catalog settings
+    #-----------------------------------------------------
+    get:
+      summary: Retrieve service catalog API settings
```

```suggestion
      summary: Retrieve Service Catalog API settings
```
Let's keep all the `Service Catalog` entries consistent with a single form.

---

Fixed in 00f53c7